libminifi/include/utils/file/FileUtils.h

```diff
@@ -248,18 +248,16 @@ inline uint64_t last_write_time(const std::string &path) {
   if (ec.value() == 0) {
     return result;
   }
-#else
-#ifdef WIN32
-  struct _stat result;
-  if (_stat(path.c_str(), &result) == 0) {
+#elif WIN32
```

Comment:
I think this should be `#elif defined(WIN32)`

Reply:
done